File: frontend/src/app/master/models/priority.ts
export interface Priority {
    // Id: number;
    // Customer: string;
    // Project: string;
    // ProjectPriority : string;
    // Status : number;
    // Remarks: string;
    id?: number;
    customerId?: string,
    projectId?:string,
    customer: string;
    project: string;
    projectPriority: string;
    remarks?: string | null;
    active?: boolean;
    status: number;
    }

File: backend/Models/Priority.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Models
{
    public class Priority:BaseModel
    {
        [Key]
        public int Id { get; set; }
        public string CustomerId { get; set; }
        public string Customer { get; set; }
        public string ProjectId { get; set; }
        public string Project { get; set; }
        public string ProjectPriority { get; set; }
        public string? Remarks { get; set; }
    }
}
